packages/ses/scripts/hermes.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+OS="$(uname -s)"
+
+case "$OS" in
+    Linux*)
+        OS_DIR="linux64-bin"
+        ;;
+    Darwin*)
+        OS_DIR="osx-bin"
+        ;;
+    CYGWIN*|MINGW*|MSYS*)
+        OS_DIR="win64-bin"
+        ;;
+    *)
+        echo "Unsupported OS: $OS"
+        exit 1
+        ;;
+esac
+
+# Paths relative to 'packages/ses'
```

Comment:
Here’s a trick to make this script insensitive to $PWD https://github.com/endojs/endo/blob/master/scripts/graph.sh#L8